### ISSUE TITLE
Enhancement: Show warning if duplicate fails to copy images

### ISSUE THREAD

**Issue by chenilim** [mattermost/focalboard#3022]
## Summary

When duplicating a board and one or more images fail to be copied (due to being missing, S3 down, etc.), show a warning to the user.

Draft message: "Warning: Some files attached to cards in the board failed to be duplicated."

## How important this is to me and why

Importance: Medium

Use cases:
1. This will alert users to potential system errors

## Additional context/similar features

Related to https://github.com/mattermost/focalboard/issues/2979.

Note that files may be missing due to being deleted or data retention. However, there may be other backend errors that could cause this.

**Comment by chenilim** [comment]
@justinegeffen - Any suggestions on improving the message? Thanks. cc @wuwinson, @asaadmahmood.

**Comment by justinegeffen** [comment]
@chenilim, is this still required? :)

**Comment by chenilim** [comment]
Yes, this is still an issue (boards can be duplicated with missing images), though not a high-priority one.
